File: recomendacao/src/benchmark-thresholds.py
from dataclasses import dataclass


@dataclass(frozen=True)
class BenchmarkThresholds:
    satisfacaoAlta: float
    satisfacaoBaixa: float
    shareAlto: float
    shareBaixo: float
    rendaAlta: float
    rendaBaixa: float
    trendUp: float
    trendDown: float


DEFAULT_BENCHMARKS = BenchmarkThresholds(
    satisfacaoAlta=7.5,
    satisfacaoBaixa=6.0,
    shareAlto=40,
    shareBaixo=30,
    rendaAlta=10000,
    rendaBaixa=3500,
    trendUp=1.0,
    trendDown=-1.0,
)

File: recomendacao/benchmark-thresholds.ts
export interface BenchmarkThresholds {
  satisfacaoAlta: number;
  satisfacaoBaixa: number;
  shareAlto: number;
  shareBaixo: number;
  rendaAlta: number;
  rendaBaixa: number;
  trendUp: number;
  trendDown: number;
}

export const DEFAULT_BENCHMARKS: BenchmarkThresholds = {
  satisfacaoAlta: 7.5,
  satisfacaoBaixa: 6.0,
  shareAlto: 40,
  shareBaixo: 30,
  rendaAlta: 10_000,
  rendaBaixa: 3_500,
  trendUp: 1.0,
  trendDown: -1.0,
};
